Plugins/Public/dronebays/Utility.cpp

```diff
@@ -0,0 +1,343 @@
+#include "Main.h"
+
+void Utility::SetRepNeutral(uint clientObj, uint targetObj)
+{
+	int targetRep;
+	int droneRep;
+	pub::SpaceObj::GetRep(targetObj, targetRep);
+	pub::SpaceObj::GetRep(clientObj, droneRep);
+
+	pub::Reputation::SetAttitude(droneRep, targetRep, 0.0f);
+
+}
+
+void Utility::SetRepHostile(uint clientObj, uint targetObj)
+{
+	int targetRep;
+	int droneRep;
+	pub::SpaceObj::GetRep(targetObj, targetRep);
+	pub::SpaceObj::GetRep(clientObj, droneRep);
+
+	pub::Reputation::SetAttitude(droneRep, targetRep, -1.0f);
+}
+
+uint Utility::CreateDroneNameInfocard(const uint& droneOwnerId)
+{
+	// Allow the plugin to use IDs range 550000 to 550500

```

Comment:
That doesn't seem to match the numbers below, but I wonder if we should have this reuse drone name infocards that are no longer in use

Reply:
.